yt_dlp/extractor/atptour.py

```diff
@@ -0,0 +1,127 @@
+import re
+
+from .brightcove import BrightcoveNewIE
+from .common import InfoExtractor
+from ..utils import base_url, extract_attributes, get_element_html_by_id, traverse_obj, urljoin
+
+
+class ATPTourVideoIE(InfoExtractor):
+    IE_NAME = 'atptour:video'
+    _VALID_URL = r'https?://(?:www\.)?atptour\.com/e[ns]/video/(?P<id>[\w-]+)'
```

Comment:
I'd use `en|es` just in case other languages get added later and following other extractors as well
```suggestion
    _VALID_URL = r'https?://(?:www\.)?atptour\.com/(en|es)/video/(?P<id>[\w-]+)'
```